pkg/containerruntime/docker.go

```diff
@@ -26,8 +26,9 @@ import (
 )
 
 const (
-	DefaultDockerVersion = "19.03"
-	LegacyDockerVersion  = "18.09"
+	DefaultDockerContainerdVersion = "1.4"
+	DefaultDockerVersion           = "19.03"
```

Comment:
too old, let's make it 20.10

Reply:
I agree we should update this, but since 1.24 is not going to use docker anymore I'm hesitant to include this change in this PR. Would you be okay with us updating it in a separate PR?